matcher_factories.go

```diff
@@ -264,13 +359,27 @@ func EqInterface(value interface{}) interface{} {
 	return nil
 }
 
+func NotEqInterface(value interface{}) interface{} {
+	RegisterMatcher(&NotEqMatcher{Value: value})
+	return nil
+}
+
 func AnyInterface() interface{} {
-	RegisterMatcher(NewAnyMatcher(reflect.TypeOf((*(interface{}))(nil)).Elem()))
+	RegisterMatcher(NewAnyMatcher(reflect.TypeOf((*interface{})(nil)).Elem()))
+	return nil
+}
+
+func EqInterfaceSlice(value []interface{}) []interface{} {
+	RegisterMatcher(&EqMatcher{Value: value})
+	return nil
+}
+
+func NotEqInterfaceSlice(value []interface{}) []interface{} {
+	RegisterMatcher(&NotEqMatcher{Value: value})
 	return nil
 }
```

Comment:
These seemed to be missing so I added them for completeness. Let me know if they had been left out for a reason.